src/block/Bell.php

```diff
@@ -152,6 +154,22 @@ public function onInteract(Item $item, int $face, Vector3 $clickVector, ?Player
 		return true;
 	}
 
+	public function onProjectileHit(Projectile $projectile, RayTraceResult $hitResult) : void{
+		$faceHit = $hitResult->getHitFace();
+		if($this->attachmentType->equals(BellAttachmentType::CEILING())){
+			$this->ring($faceHit);
+		}
+		if($this->attachmentType->equals(BellAttachmentType::FLOOR()) && Facing::axis($faceHit) === Facing::axis($this->facing)){
+			$this->ring($faceHit);
+		}
+		if(
```

Comment:
same again here

Reply:
In fact all of these conditions can be merged, since the bodies of all of them are identical.